android/gradle.properties

```diff
@@ -25,10 +25,10 @@ kotlinVersion=1.3.50
 minSdkVersion=23
 compileSdkVersion=31
 targetSdkVersion=31
-buildToolsVersion=31.0.0
+buildToolsVersion=30.0.0
```

Comment:
I am curious about the downgrade in the build tools version

Reply:
I was trying to follow the guidelines here : https://react-native-community.github.io/upgrade-helper/?from=0.64.3&to=0.67.5

Reply:
I'll try bumping it back to 31 and see if we get a successful build :)

Reply:
Hmm okay. But I see the `compileSdkVersion` and `targetSdkVersion` do not match what's in there. And build tools version used is `30.0.2`. Also, I think we are upgrading from `0.63.5`.